src/components/OrderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import {ThemeProvider} from '../ThemeProvider';
+import {Card} from '../Card';
+import {TextContainer} from '../TextContainer';
+import {Thumbnail} from '../Thumbnail';
+import {Stack} from '../Stack';
+import {Button} from '../Button';
+import {TextStyle} from '../TextStyle';
+
+export interface OrderDetailsProps {}
+
+export class OrderDetails extends React.PureComponent<OrderDetailsProps> {
+  render() {
+    return (
+      <ThemeProvider
+        theme={{
+          colorScheme: 'dark',
+          colors: {
+            surface: '#020c1d',
+            secondary: '#103560',
+            interactive: '#009973',
+          },
+        }}
+      >
+        <Card title="Order details">
```

Comment:
In production, all of these texts would be translated via the [`i18n` tool](https://github.com/Shopify/quilt/tree/master/packages/react-i18n). 

```
const [i18n] = useI18n();

<Card title={i18n.translate('OrderDetails.heading')}>
```